agent/lib_phpunit.c

```diff
@@ -665,6 +666,22 @@ static int nr_phpunit_are_statuses_valid(TSRMLS_D) {
   return 1;
 }
 
+void nr_phpunit_version(TSRMLS_D) {
+  char* string = "PHPUnit\\Runner\\Version::id();";
+  zval retval;
+  int result
+      = zend_eval_string(string, &retval, "Retrieve PHPUnit Version" TSRMLS_CC);
+
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      // Add php package to transaction
+      nr_txn_add_php_package(NRPRG(txn), "phpunit/phpunit", version);
+      zval_dtor(&retval);
+    }
+  }
+}
```

Comment:
Would it make sense to add phpunit w/o a version if the version is not detected?

Reply:
Yes, thanks for catching that. Fixed: [3eaa90f](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/3eaa90f72f9976ab895574749ae180d1acad0c0a)